net-proxy/dante/dante-1.4.1-r6.ebuild

```
@@ -48,6 +48,10 @@ PATCHES=(
 src_prepare() {
 	default
 
+	# 780039
+	sed -e 's/-all-dynamic//' -i dlib/Makefile.am
```

|| die on external commands which may fail, also we can just give multiple file arguments:
```suggestion
	sed -e 's/-all-dynamic//' -i dlib/Makefile.am dlib64/Makefile.am || die
```

---

Thanks, that is better.